src/dmd/dsymbol.d

```diff
@@ -1180,6 +1180,11 @@ extern (C++) class Dsymbol : RootObject
         return null;
     }
 
+    inout(CtorDeclaration) isCopyCtorDeclaration() inout
```

Comment:
This is unlike all the other `isXXXDeclaration()` is copy constructors are not a distinct object. It's a flag in the CtorDeclaration(). So it's confusing. Also, there's only one use of the virtual-ness of this, so this function can be discarded and replaced with isCtorDeclaration() && isCopyCtor.